### DISCUSSION TITLE
Status of this project / what needs to be done

### DISCUSSION THREAD

**Discussion by ghost** [minexew/Shrine#59] [General]
I need an environment to do programming in holy C while still being able to access the internet and do normal things. What needs to be done on here / what's the status? Willing to contribute. Zenith doesn't have internet access, so I think it's better to use Shrine. Can you run TeX / emacs on here yet?

**Comment by minexew** [comment]
> Can you run TeX / emacs on here yet?

Sir, this is TempleOS
